src/generated/api/servicecontrol/v1/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Code generated by sidekick. DO NOT EDIT.
+
+[package]
+name                 = "google-cloud-api-servicecontrol-v1"
```

Comment:
This is fine, but if we decide to flatten out the usage of devtools from crate naming we should do the same for this directory as well.

Reply:
Yup.  There is also `google/cloud/privacy/dlp` (see #917).  I think the question is how many of these directories with multiple services we have.  We can use manual overrides if there are just a handful.  If there are "lots" then we may want to add some code to handle them.